packages/grid/_modules_/grid/components/panel/GridPanel.tsx

```diff
@@ -82,12 +85,22 @@ const GridPanel = React.forwardRef<HTMLDivElement, GridPanelProps>((props, ref)
           name: 'flip',
           enabled: false,
         },
+        {
+          name: 'isPlaced',
+          enabled: true,
+          phase: 'main',
+          fn: ({ state }) => {
+            if (state.placement === GRID_PANEL_POSITION) {
```

Comment:
If the placement is not `bottom-start` the filter panel content is never visible. Could we make it to work on any placement?

```suggestion
```